docs/smart_contract_sdk/build_deploy_contract.md

```diff
@@ -73,6 +76,26 @@ Run the script by providing your compiled contract file path.
 ```
 The output file `optimized-<ORIGINAL_FILENAME>.wasm` is generated under the same location of your script.
 
+### Windows 10 
+
+We recommend that you use this optimization commands, otherwise you will not have enough tokens to run a contract. The faucet gives enough tokens to run contracts that have been built with the optimized commands.
+
+In your terminal, type the following commands:
```

Comment:
These are commands for linux and will not work in Windows 10. Linux is a unix-like system so the commands above is applicable. Windows 10's command line set belongs to "DOS, OS/2" family. You can emulate the steps above by install "Windows subsystem for linux(WSL) " to emulate a linux shell in windows. See [https://docs.microsoft.com/en-us/windows/wsl/about](https://docs.microsoft.com/en-us/windows/wsl/about)

We will be pushing an update for both the windows and linux script.

Thanks again for the feedback and please let us know what else could be made clearer in the future